objects/oBoss1Head.object.gmx

```diff
@@ -63,6 +62,63 @@ wall2.image_yscale = 3;
 wheel = instance_create(x, y, oBoss1Wheel);
 
 sparkcounter = 0;
+
+//~TIMING~//
+enum BOSS1_ATTACKTIME {
+    INITIAL = 120,
+    NORMAL = 90
+}
+
+
+//these didn't work right after the refactor
+//I think I made him too efficient lol
+/*
+
+enum BOSS1_INTRO { //old times
+    MoveTiles0 = 60,
+    MoveTiles1 = 92,
+    PlayMusic = 90,
+    FadeIn = 120,
+    ScanEvent = 150,
+    Quake = 210,
+    Animate = 220,
+    ToNormal = 280
+}
+
+*/
+
+enum BOSS1_INTRO {
+    MOVETILES0 = 60,
+    MOVETILES1 = 92,
+    PLAYMUSIC = 90,
+    FADEIN = 120,
+    SCANEVENT = 150,
+    QUAKE = 160,
+    ANIMATE = 180,
+    TONORMAL = 200
+}
+
+enum BOSS1_ATTACK {
+    EYEFLASH = 5,
+    ATTACK = 80,
+    TONORMAL = 120
+}
+
+enum BOSS1_DEATH {
+    BURN = 1,
+    EXPLODE = 250
+}
+
+
+//~PHASE PROGRESSION ARRAY~//
+//hp at each phase
+PhaseProgression[0] = 600;
```

Comment:
Please refer to contribution guidelines for constants. Also: shouldn't these pull from the HP mods in modifiers.ini?

Reply:
These were hard-coded values. If it was to pull from modifiers.ini, I would've had this array do that